http-api/src/lib.rs

```diff
@@ -46,14 +46,25 @@ pub struct Context {
 /// Run the HTTP API.
 pub async fn run(options: Options) {
     let paths = Paths::from_root(options.root).unwrap();
+    let storage = ReadOnly::open(&paths).expect("failed to read storage paths");
+    let peer_id = storage.peer_id().to_owned();
+
     let ctx = Context {
         paths,
         theme: options.theme,
     };
+    // let peer_id = peer_id_from_identity(options.identity, options.identity_passphrase)
+    //     .expect("--identity path flag is missing");
+
+    let peer_id_route = warp::path("peer-id")
+        .and(warp::get())
+        .and_then(move || get_peer_id(peer_id));
+
     let api = path("v1")
         .and(path("projects"))
         .and(filters(ctx))
         .or(warp::get().and(path::end()).and_then(root_handler))
+        .or(peer_id_route)
```

Comment:
Isn't this going to put it under `/v1/projects/peer-id`?

Reply:
thanks, good catch. Commit b7358c9 re-organizes. Although want to confirm that `/v1/projects` should return 404. Seems to be doing this originally in master, but wanted to double check that this is indeed the expected case.